### DISCUSSION TITLE
can i create a egde without target node?

### DISCUSSION THREAD

**Discussion by PeanutQAQ** [xyflow/xyflow#3157] [General]
hi guys. i have a request that i should create a edge which have a source but do not have a target. i look for answer in docs but find nothing. someone can help QvQ

**Comment by bcakmakoglu** [comment]
No, you can't.
What you *can* do though, is create an "invisible" target node for the edge to make it look like it doesn't have a target node at all.
Something similar is used in the [floating edges example](https://reactflow.dev/docs/examples/edges/floating-edges/).

**Reply by PeanutQAQ** [reply]
Thanks for your reply, I ended up using a custom node to simulate the edge style in order not to pollute the node data.
Again, thanks for this awesome project!
